tests/unit/restapi/lib/actions.py

```diff
@@ -101,3 +104,32 @@ def register_queue(
         json=payload,
         follow_redirects=True,
     )
+
+
+def create_plugin_parameter_type(
```

Comment:
```suggestion
def register_plugin_parameter_type(
```
Change create to register to keep established naming schema.